slave.mk

```diff
@@ -421,6 +421,8 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : .platform
 		chmod +x sonic_debian_extension.sh,
 	)
 
+	DIRTY_SUFFIX="$(shell date +%s)"
```

Comment:
date +%s = 1505491407 unreadable?

can we get a more readable format. like YYYYMMDD.HHMMSS